po/cs.po

```diff
@@ -0,0 +1,510 @@
+msgid "Archive"
+msgstr "Archivovat"
+
+#: src/ui/project/project_edit_window.ui:118 src/ui/project/task_row.ui:124
+msgid "Delete"
+msgstr "Smazat"
+
+#: src/ui/project/project_header.ui:92
+msgid "Time spent per day"
+msgstr "Čas strávený denně"
+
+#: src/ui/project/project_list.ui:82 src/ui/project/task_page.ui:146
+msgid "Use add button up there"
+msgstr "Pro nový úkol použijte tlačítko plus"
+
+#: src/ui/project/project_list.ui:119
+msgid "Done tasks"
+msgstr "Hotové úkoly"
+
+#: src/ui/project/project_list.ui:149
+msgid "Delete list"
+msgstr "Smazat seznam"
+
+#: src/ui/project/project_lists.ui:35
+msgid "no lists found"
+msgstr "nenalezeny žádné seznamy"
+
+#: src/ui/project/project_lists.ui:64
+msgid "New list"
+msgstr "Nový seznam"
+
+#: src/ui/project/record_window.ui:16
+msgid "New Record"
+msgstr "Nový záznam"
+
+#: src/ui/project/record_window.ui:24 src/ui/project/reminder_window.ui:24
+msgid "Cancel"
+msgstr "Zrušit"
+
+#: src/ui/project/record_window.ui:30 src/ui/project/reminder_window.ui:30
+msgid "Done"
+msgstr "Hotovo"
+
+#: src/ui/project/record_window.ui:47
+msgid "Starts"
+msgstr "Začíná"
+
+#: src/ui/project/record_window.ui:50 src/ui/project/record_window.ui:70
+#: src/ui/project/reminder_window.ui:49
+msgid "Date"
+msgstr "Datum"
+
+#: src/ui/project/record_window.ui:57 src/ui/project/record_window.ui:77
+#: src/ui/project/reminder_window.ui:55
+msgid "Time"
+msgstr "Čas"
+
+#: src/ui/project/record_window.ui:67
+msgid "Ends"
+msgstr "Končí"
+
+#: src/ui/project/record_window.ui:86
+msgid "Duration"
+msgstr "Doba trvání"
+
+#: src/ui/project/record_window.ui:98 src/ui/project/reminder_window.ui:66
+msgid "delete"
+msgstr "smazat"
+
+#: src/ui/project/reminder_window.ui:16
+msgid "Reminder"
+msgstr "Upomínka"
+
+#: src/ui/project/task_page.ui:30
+msgid "Reminders"
+msgstr "Upomínky"
+
+#: src/ui/project/task_page.ui:92
+msgid "Records"
+msgstr "Záznamy"
+
+#: src/ui/project/task_page.ui:145
+msgid "no subtasks found"
+msgstr "nebyly nalezeny žádné dílčí úkoly"
+
+#: src/ui/project/task_page.ui:178
+msgid "no records found"
+msgstr "nebyly nalezeny žádné záznamy"
+
+#: src/ui/project/task_page.ui:179
+msgid "Use add button up there or start a timer"
+msgstr "Použijte tlačítko Přidat nebo spusťte časovač"
+
+#: src/ui/project/task_row.ui:109 src/views/project/task_row.rs:157
+msgid "Timer"
+msgstr "Časovač"
+
+#: src/ui/search/search_window.ui:16
+msgid "Search"
+msgstr "Vyhledávání"
+
+#: src/ui/search/search_window.ui:58
+msgid "no results found"
+msgstr "nebyly nalezeny žádné výsledky"
+
+#: src/ui/sidebar/sidebar_projects.ui:14
+msgid "Projects"
+msgstr "Projekty"
+
+#: src/ui/sidebar/sidebar_projects.ui:33
+msgid "Show archive projects"
+msgstr "Zobrazit archivní projekty"
+
+#: src/ui/sidebar/sidebar_projects.ui:40
+msgid "New project"
+msgstr "Nový projekt"
+
+#. Translators: Replace "translator-credits" with your names, one name per line
+#: src/application.rs:196
+msgid "translator-credits"
+msgstr "Amerey.eu"
```

Comment:
You can use these formats:
- email: `Your name <example@gmail.com>`
- website: `Your name https://www.amerey.eu`

Reply:
By adding URL with this format `Amerey.eu https://amerey.eu` users able to open the link by clicking on that.